JSDemos/Demos/List/ListEditingAndAPI/description.md

```diff
@@ -1 +1,23 @@
-The **List** is&nbsp;a&nbsp;UI component that represents a&nbsp;collection of&nbsp;items in&nbsp;a&nbsp;scrollable list. This demo displays different delete modes and shows how you can handle the **selectionChanged** event.
\ No newline at end of file
+The [List](/Documentation/ApiReference/UI_Components/dxList/) displays a collection of items as a scrollable list. To create a list, pass an array of items to the [dataSource](/Documentation/ApiReference/UI_Components/dxList/Configuration/#dataSource) property. 
```

Comment:
It may be better to keep this a text as this demo is the first in the series. This text will serve as an introduction to the List demo series as well as a reminder to users as to how to create a list.